tests/karate/src/test/java/fe/FrontEndTest.java

```diff
@@ -10,32 +10,6 @@
 
 public class FrontEndTest {
 
-  /**
-   * This test will execute all front-end tests with the web front-end
-   * It will not generate a clean report, but it can be used in development
-   *
-   * @return the Karate builder
-   */
-  @Karate.Test
-  Karate testWeb() {
```

Comment:
Aren't these functions useful for development?

Reply:
Yes, I put them back.